File: 242.go
package main

import "sort"

func isAnagram(s string, t string) bool {

	if len(s) != len(t) {
		return false
	}

	return sort.Sort(s) == sort.Sort(t)
}

func main() {
	println(isAnaragram("cat", "tac"))
}

File: 242.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
  s = s.split('').sort().join('')
  t = t.split('').sort().join('')
  return s === t;
};

var s = "anagram", t = "nagaram"
console.log(isAnagram(s, t));
var s = "rat", t = "car"
console.log(isAnagram(s, t));